.github/workflows/build-images-beta.yaml

```diff
@@ -18,6 +18,17 @@ permissions:
   id-token: write
 
 jobs:
+  add-inputs-to-env:
+    if: ${{ github.event_name == "workflow_dispatch" }}
+    name: Add Workflow Dispatch Inputs to Env
+    environment: release-beta-images
+    runs-on: ubuntu-22.04
+    env:
+      INPUTS: ${{ tojson(inputs) }}
+    steps:
+      - run: |
```

Comment:
Steps also require a name, validation is failing https://github.com/cilium/cilium/actions/runs/8438843502/job/23112017407?pr=31424

Reply:
Ah I didn't know we had this kind of validation, thanks!